SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Channel/Channel.cs

```diff
@@ -43,12 +43,11 @@ public sealed class Channel : IChannelUnit
             _batchScheduled = false;
             _enabled = true;
             DeviceInformationHelper.InformationInvalidated += (sender, e) => InvalidateDeviceCache();
-            Task.Run(CountFromDiskAsync);
```

Comment:
I know my comment last time asked if we could delete this, but looking now I guess we actually do need it? Problem is that we don't necessarily call SetEnabledAsync(true) before using, so either we should (but can't in constructor) or we should restore this